src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,75 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final Random random = new Random();
+    private static final int MAX_INT = 25;
+    private static final ColorSupplier colorSupplier = new ColorSupplier();
+    private static final String[] FIGURES = {"Rectangle",
```

Comment:
remove this array, you don't need it, just using Random generate number from 0 to number of figures, based on this number choose a figure

Reply:
Array is removed.
Random generator of number from 1 to number of figures in use